packages/tools/generator-widget/generators/app/index.js

```diff
@@ -61,7 +61,7 @@ class MxGenerator extends Generator {
 
     install() {
         this.log(text.INSTALL_FINISH_MSG);
-        this.npmInstall();
+        this.npmInstall(undefined, { legacyPeerDeps: true });
```

Comment:
I'm not entirely sure if this is a CLI script, but maybe it's an improvement to make this a command flag that defaults to `true` so users can also opt out? Or is that not relevant for this?

Reply:
It's not relevant. NPM versions below v7 will ignore this flag. For v7 or higher it's always required. When PIW tools dependencies get aligned we can remove this argument again.

Reply:
okay, clear 👍